src/coreComponents/physicsSolvers/contact/SolidMechanicsEmbeddedFractures.cpp

```diff
@@ -664,18 +664,22 @@ void SolidMechanicsEmbeddedFractures::applySystemSolution( DofManager const & do
     updateJump( dofManager, domain );
   }
 
-
   forMeshTargets( domain.getMeshBodies(), [&] ( string const &,
                                                 MeshLevel & mesh,
                                                 arrayView1d< string const > const & )
   {
-    std::map< string, string_array > fieldNames;
-    fieldNames["elems"].emplace_back( string( extrinsicMeshData::contact::dispJump::key() ) );
-    fieldNames["elems"].emplace_back( string( extrinsicMeshData::contact::deltaDispJump::key()) );
-    CommunicationTools::getInstance().synchronizeFields( fieldNames,
-                                                         mesh,
-                                                         domain.getNeighbors(),
-                                                         true );
+    std::vector< SyncFieldsID > fieldsTobeSync;
+    array1d< string > regionNames, fieldNames;
+    regionNames.emplace_back( getFractureRegionName() );
+    fieldNames.emplace_back( extrinsicMeshData::contact::dispJump::key() );
+    fieldNames.emplace_back( extrinsicMeshData::contact::deltaDispJump::key() );
+
+    fieldsTobeSync.emplace_back( SyncFieldsID{ FieldLocation::Elem, regionNames, fieldNames } );
+
+    CommunicationTools::getInstance().synchronizeFields2( fieldsTobeSync,
```

Comment:
In this precise case, this means that we'll always provide a `std::vector< SyncFieldsID >` with one unique element.
Do you think this will this always be the case? If so, we could simplify `synchronizeFields2` such that it receives a unique instance of `SyncFieldsID` and not a `std::vector`?

Reply:
I think having a vector can be useful. If, for example, we had a solver with a mixed formulation (e.g., velocity unknowns at the faces &  pressures at the cell centres) one may want to pass two different fields. The same could occur if the same solver had specific variables for different regions. I am not sure we have a use case right now (maybe the hybridFVM?) but I do think it's useful.